packages/gatsby-plugin-styled-components/src/gatsby-node.js

```diff
@@ -4,30 +4,15 @@ try {
   require.resolve(`babel-plugin-styled-components`)
   babelPluginExists = true
 } catch (e) {
-  // Ignore
+  throw new Error(`'babel-plugin-styled-components' is not installed`)
```

Comment:
I don't why we should install `babel-plugin-styled-components` manually now, what's the concern for this change? also if I throw here, `babelPluginExists` is not necessary actually